### ISSUE TITLE
`accounts` is not a heroku command.

### ISSUE THREAD

**Issue by andela-idurotola** [ddollar/heroku-accounts#86]
This is what I get after adding my private keys to each account

**Comment by JangoSteve** [comment]
Getting the same.

**Comment by JangoSteve** [comment]
Just figured out what happened, the instructions for this don't exactly happen where you think they would, which is why I missed this. When you install this plugin, it installs fine, and the first time you run it, it will actually run that command fine as well, but then it automatically uninstalls the plugin after that with this output message:

```
$ heroku accounts:add some-account --auto
Uninstalling deprecated ddollar/heroku-accounts plugin...Done. Use https://github.com/heroku/heroku-accounts instead.Enter your Heroku credentials.
Email:
```

The first time I did this, I saw the email field and logged in and then everything stopped working. I didn't realize that before it successfully ran the first command it gave some output message at the top that said it was *also* uninstalling the plugin (which of course then leads to the message that `accounts` is not a heroku command).

Installing the plugin from the heroku repo made it work again:

```
heroku plugins:install https://github.com/heroku/heroku-accounts
```
